appservice/src/createAppService/AppServicePlanSkuStep.ts

```diff
@@ -32,15 +32,16 @@ export class AppServicePlanSkuStep extends AzureWizardPromptStep<IAppServiceWiza
             return {
                 label: s.label || nonNullProp(s, 'name'),
                 description: s.description || s.tier,
-                data: s
+                data: s,
+                group: s.group || wizardContext.advancedCreation ? localize('additionalOptionsLabel', 'Additional Options') : ''
```

Comment:
I don't think you need to check `advancedCreation` here since enableGrouping will be set to false below anyways